courses/management/commands/gen_realish_data.py

```diff
@@ -0,0 +1,64 @@
+"""
+Generates fake data
+"""
+from random import randint
+from django.core.management import BaseCommand
+
+from courses.models import Course
+from courses.factories import CourseFactory, ModuleFactory
+from oauth_mgmt.factories import BackingInstanceFactory
+from oauth_mgmt.models import BackingInstance
+
+
+class Command(BaseCommand):
+    """
+    Generates fake data
+    """
+    help = "Generates data that looks real, but isn't"
+
+    def handle(self, *args, **options):
+        try:
+            bi = BackingInstance.objects.get(instance_url='https://edx.org')
+        except BackingInstance.DoesNotExist:
+            bi = BackingInstanceFactory.create(instance_url='https://edx.org')
+
+        course_title = 'Advanced Introductory Classical Mechanics'
+        try:
+            c = Course.objects.get(title=course_title)
+            for m in c.module_set.all():
+                m.delete()
+
+        except Course.DoesNotExist:
+            c = CourseFactory.create(
+                edx_instance = bi,
+                title=course_title,
+                author_name='David E. Prichard',
+                description='Mechanics ReView presents a college-level introductory mechanics class using a strategic problem-solving approach.',
+                overview='For more information, see <a href="#">edx.org</a>',
+                image_url='https://www.edx.org/sites/default/files/styles/course_video_banner/public/course/image/featured-card/8.05.3x-video_still-318x210.jpg?itok=DJOP5xRS',
```

Comment:
This image goes with this course:

https://www.edx.org/sites/default/files/styles/course_video_banner/public/course/image/featured-card/8.mechcx-378x225.jpg?itok=lDOLQ61K